src/App.scss

```diff
@@ -20,51 +19,63 @@ body {
 }
 
 h1 {
```

Comment:
You should probably change this to a class, just in case you add other `<h1>`'s in the future or if you want to add shared styling between text elements. Name it something like `.title`.

Reply:
You might also want to consider adding a class to the `<header>` and handling the layout styling there instead of the `<h1>` just in case you need to add other elements to the header down the line. Something like might work:

```
.header {
    border-bottom: $grey-border;
    display: flex;
    flex-direction: column;
    padding: 1.5rem 0rem;
    place-items: center center;
    
    .title {
        font-size: 3rem;
    }
}
```